src/components/Collection/Collection.tsx

```diff
@@ -49,25 +50,22 @@ const Collection: React.FC<{
 
   if (!collectionResult) return <Loading />;
 
-  const collectionId = CollectionUtils.getId(collectionResult.links);
-
-  const collectionMetadata = collectionResult.metadata;
-  const itemsPerPage = collectionMetadata.itemsPerPage;
-  const totalItems = collectionMetadata.numberOfItems;
-  const lastPageLink =
-    collectionResult.links[collectionResult.links.length - 1].href;
+  const { metadata, links, publications } = collectionResult;
+  const { itemsPerPage, numberOfItems, currentPage, title, description } =
+    metadata;
```

Comment:
object destructuring here to help us write less `metadata.title`, `metatada.description`, etc. later in the component